### PR TITLE
ci(docs): deploy to deno

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

**Comment by yinonov** [comment]
@YonatanKra can u please remove the - now redundant - production build of docs?

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -30,4 +30,4 @@ jobs:
   # ! temporary. site deployment should move to latest release
   call-deploy-site:
```
Comment (YonatanKra) [inline .github/workflows/pre-release.yml:31]: This should be in the `release` right after the `publish` step.

Comment (yinonov) [reply .github/workflows/pre-release.yml:31]: thinking again, why? this workflow is called pre-release, meaning it handles the release flow.
We have a synchronous workflow where 'publish-package' is followed by 'deploy' which declaratively prerequisite it

Comment (YonatanKra) [reply .github/workflows/pre-release.yml:31]: Oh - ok. So it is not `pre-release`, it is the `release`. 
The filename is misleading...

**Review by YonatanKra — APPROVED** [review]
Maybe change the `pre-release.yml` filename to `release-vivid.yml` or something?

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**